servicecontrol/servicecontrol-instances/hardware.md

```diff
@@ -9,7 +9,8 @@ ServiceControl as an application can be used to process the entire message load
 ## General recommendations
 
 * Install ServiceControl on a dedicated server in production.
-* 6 GB of RAM minimum
+* Do not host ServiceControl and ServiceControl.Audit instance on the same server, host in isolation
```

Comment:
I don't know if this is applicable for customers with low traffic/load?

Reply:
I've rewritten that part.